contracts/tasks/staking_upgrade.ts

```diff
@@ -9,6 +9,7 @@ import { ethers } from "ethers"
 import { assertContractVariable, getContractAddressByName, awaitCondition, storage } from "../src/deploy-utils"
 import { ImplStorageName, ProxyStorageName, ContractFactoryName } from "../src/types"
 import { predeploys } from "../src"
+import { hexlify } from "ethers/lib/utils";
 
 task("rollup-deploy-init")
     .addParam("storagepath")
```

Comment:
should remove this param?

Reply:
const bytecode = hexlify(...../plonk_verifier_0.10.3.bin);